### ISSUE TITLE
Status on php 7 support?

### ISSUE THREAD

**Issue by w3l** [klarna/kco_php#14]
The klarna [docs](https://developers.klarna.com/en/se/kco-v2/download/php) claims:
> PHP >= 5.x (PHP 7 is currently not supported)

[kco_php](https://github.com/klarna/kco_php) seems to support PHP 7, based on our internal tests.

* No issues found testing the SDK code with [alexia/php7mar](https://packagist.org/packages/alexia/php7mar) and [sstalle/php7cc](https://packagist.org/packages/sstalle/php7cc).
* No PHP 7 issues with forked code in travis(disclaimer: I do not fully understand Travis/GIT).
* Everything works as expected running Checkout on our dev-site again testdrive(all snippets rendering, we get the push, no errors in our log-files, etc)
* Everything works as expected running Checkout on our live-site again klarnas live system(all snippets rendering, we get the push, no errors in our log-files, etc)

The reply contacting integration@klarna.com was basically(paraphrasing a bit):
> It might work under php 7, but we do not support php 7.
> It has not been prioritized(to test php 7 support), so we do not know if it works or not.

I assumed that the lack of support for php 7 was due to either major code rewrites needed or that it would break compatibility with earlier php 5.x versions. Not wanting to test the code was an unexpected and a bit disappointing reason.

It would feel a lot better to get official php 7 support in klarna checkout before we switch php version on our live server.

**Has anyone in the community run into any issues with using Klarna Checkout on PHP 7?**

**Comment by w3l** [comment]
Update: We have been running [kco_php](https://github.com/klarna/kco_php) on php 7 for a couple of months with no issues.

**Comment by sjelfull** [comment]
Yeah, it's pretty dissapointing that they obviously does not care to keep this maintained.

**Comment by mladenilic** [comment]
Seems like new website design took priority over working SDK. ;-)
